Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[workspace]
+members = ["db", "cli"]
```

Comment:
Cargo Workspaces allow you to have multiple crates in the same directory and tie them together a bit! This says that the `db` and `cli` sub-directories here are crates we want to include in our Workspace